packages/app/app/containers/HttpApiUrl/index.js

```diff
@@ -15,7 +15,6 @@ const HttpApiUrl = ({ settings }) => {
   return (
     <a
       className={styles.http_api_url}
-      href='javascript:void(0)'
```

Comment:
I think this href was needed for some cosmetic reason, did you notice anything different when removing it?

Reply:
i replace it with css style 'cursor: pointer;'. it remove the annoying warning in console

Reply:
I replace it with css style
```css
cursor: pointer;
```
it remove the annoying warning